jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/impl/model/xml/JaxbTask.java

```diff
@@ -58,7 +58,7 @@
 
 @XmlRootElement(name="task")
 @XmlAccessorType(XmlAccessType.FIELD)
-@JsonIgnoreProperties({"archived","deadlines"})
```

Comment:
This was changed to just ignore the deadlines attribute, and allow the archived attribute to be passed now